### DISCUSSION TITLE
Spending money from OpenCollective?

### DISCUSSION THREAD

**Discussion by gshpychka** [nvim-treesitter/nvim-treesitter#7353] [General]
Greetings. I wanted to contribute to the project on OpenCollective over at https://opencollective.com/nvim-treesitter, but I noticed that no money from it has been spent yet. 

Are there plans for the money? I [asked the question](https://opencollective.com/nvim-treesitter/conversations/any-plans-for-the-money-dpoby7xq) on OpenCollective, but got no answer.

**Comment by clason** [comment]
Thank you, but we no longer accept donations as the OpenCollective admins have moved on. I'll remove the sponsorship link; thanks for the reminder.

We don't need money, we need help ;)

**Reply by gshpychka** [reply]
Whoa, what does that mean for the money that's already collected? And there are 6 people that have recurring donations set up as of right now.

**Reply by clason** [reply]
I will have to ask the admins (previous active maintainers). The way we discussed things, recurring donations should have been canceled, and it seems they forgot to stop that.

As I don't have access to the money, I cannot speak to its use; one possibility would be to donate it further upstream (Neovim or tree-sitter).

I can assure you that the money will not "go walkabout" and will only benefit entities related to nvim-treesitter.

**Comment by clason** [comment]
ping @vigoux @theHamsta @steelsojka @kyazdani42 @stsewd

**Reply by kyazdani42** [reply]
To be honest, i don't see what nvim-treesitter plugin could do with money. What I would think, is to donate what we have to neovim directly, since they need more things than us (building servers, remunerating long time developers...). 

What do you all think ? pinging again @vigoux @theHamsta @steelsojka @stsewd

**Reply by stsewd** [reply]
Hi, sorry I missed this! From my part, I'm also +1 on donating the money to neovim or `:h uganda`.

**Reply by clason** [reply]
Important would be to cancel all recurring donations and to stop accepting new ones -- the sooner, the better!

**Reply by stsewd** [reply]
Doesn't look like I have permissions to do so, probably @vigoux has.

**Reply by clason** [reply]
@kyazdani42 should, too

**Reply by gshpychka** [reply]
My expectation when donating would be that the money would be distributed to the contributors. Donating straight to neovim and shutting down the collective works, of course.
